ui/lib/pki/addon/routes/configuration/tidy.js

```diff
@@ -4,5 +4,31 @@
  */
 
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import { hash } from 'rsvp';
+import { withConfirmLeave } from 'core/decorators/confirm-leave';
 
-export default class PkiConfigurationTidyRoute extends Route {}
+@withConfirmLeave('model.tidy')
+export default class PkiConfigurationTidyRoute extends Route {
+  @service store;
+  @service secretMountPath;
+
+  model() {
+    const backend = this.secretMountPath.currentPath;
+
+    return hash({
```

Comment:
Do we need `engine` here? I didn't see it used anywhere. If that's the case, then we can just return the record here: 
```suggestion
    return this.store.createRecord('pki/tidy', { backend })
```